data/src/test/java/com/example/data/mappers/WeatherDataEntityMapperTest.kt

```diff
@@ -0,0 +1,113 @@
+package com.example.data.mappers
+
+import com.example.data.db.tables.WeatherDataEntity
+import com.example.data.modelsApi.currentWeather.CurrentWeatherApiModel
+import com.example.data.modelsApi.currentWeather.Data
+import com.example.data.modelsApi.currentWeather.Weather
+import com.example.data.modelsApi.multiDaysWeather.DailyData
+import com.example.data.modelsApi.multiDaysWeather.DailyWeather
+import com.example.data.modelsApi.multiDaysWeather.DailyWeatherApi
+import com.example.data.utils.CityConverter
+import com.example.domain.models.CityModel
+import com.example.domain.models.WeatherData
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.mockito.Mockito.`when`
+import org.mockito.Mockito.mock
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+internal class WeatherDataEntityMapperTest {
+
+    private val cityConverter = mock(CityConverter::class.java)
+    private val weatherDataEntityMapper = WeatherDataEntityMapper(cityConverter)
+    private val iconUrl = "https://www.weatherbit.io/static/img/icons/"
```

Comment:
Use constant ICON_URL that is used in WeatherDataEntityMapper class.

Reply:
Fixed